extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml

```diff
@@ -161,38 +161,6 @@
               </field>
             </object>
           </object-param><object-param>
-            <name>development</name>
```

Comment:
OK, and what about the portlet/gadget catalog ? Is it still usable after the migration ?